packages/grid/x-data-grid-premium/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
-    "types": ["react", "mocha", "node"],
-    "noImplicitAny": true
+    "types": ["react", "mocha", "node"]
```

Comment:
I wanted to set the `"strict": true` flag for the packages that have `"noImplicitAny": true`, but because [`noImplicitAny` was disabled at the root level](https://github.com/mui/mui-x/blob/22bf0baba0167553070d3f9a5b842a27505296cd/tsconfig.json#L8), I still had to keep `"noImplicitAny": true` there, so it's not overridden by the root-level tsconfig.

I decided to make the `strict` flag opt-out instead of opt-in and enabled it in the root-level tsconfig.json.